code/modules/hydroponics/grown/towercap.dm

```diff
@@ -78,33 +78,29 @@
 
 	return NONE
 
-/obj/item/grown/log/attackby(obj/item/W, mob/user, params)
-	if(W.get_sharpness())
-		user.show_message(span_notice("You make [plank_name] out of \the [src]!"), MSG_VISUAL)
-		var/seed_modifier = 0
+/obj/item/grown/log/attackby(obj/item/attacking_item, mob/user, params)
+	if(attacking_item.get_sharpness())
+		var/plank_count = 1
 		if(seed)
-			seed_modifier = round(seed.potency / 25)
-		var/obj/item/stack/plank = new plank_type(user.loc, 1 + seed_modifier, FALSE)
-		var/old_plank_amount = plank.amount
-		for (var/obj/item/stack/ST in user.loc)
-			if (ST != plank && istype(ST, plank_type) && ST.amount < ST.max_amount)
-				ST.attackby(plank, user) //we try to transfer all old unfinished stacks to the new stack we created.
-		if (plank.amount > old_plank_amount)
-			to_chat(user, span_notice("You add the newly-formed [plank_name] to the stack. It now contains [plank.amount] [plank_name]."))
+			plank_count += round(seed.potency / 25)
+
+		user.balloon_alert(user, "made [plank_count] [plank_name]")
+		new plank_type(user.loc, plank_count)
 		qdel(src)
+		return
 
-	if(CheckAccepted(W))
-		var/obj/item/food/grown/leaf = W
+	if(CheckAccepted(attacking_item))
+		var/obj/item/food/grown/leaf = attacking_item
 		if(HAS_TRAIT(leaf, TRAIT_DRIED))
-			user.show_message(span_notice("You wrap \the [W] around the log, turning it into a torch!"))
-			var/obj/item/flashlight/flare/torch/T = new /obj/item/flashlight/flare/torch(user.loc)
-			usr.dropItemToGround(W)
-			usr.put_in_active_hand(T)
+			user.balloon_alert(user, "torch crafted")
```

Comment:
Not even totally sure this is necessary- it doesn't provide more feedback than seeing that you now have a torch in your hands.

Reply:
I don't entirely disagree with you, but at the same time, I think it's good to have two sources of feedback when making something instead of just one. Otherwise we could realistically apply the same logic to cutting planks or pulling cotton, you already see things disappearing on your screen, it's just nice to have a little confirmation that something did in fact happen.